.circleci/Dockerfile

```diff
@@ -0,0 +1,77 @@
+FROM debian:jessie as itms_installer
+
+ENV ITMS_VERSION "1.9.7"
+USER root
+
+WORKDIR /tmp
+
+# Install iTMSTransporter
+ADD https://github.com/janpio/fastlane-ubuntu-foo/raw/master/iTMSTransporter_installer_linux_$ITMS_VERSION.sh .
+RUN chmod +rx ./iTMSTransporter_installer_linux_$ITMS_VERSION.sh \
+	&& ./iTMSTransporter_installer_linux_$ITMS_VERSION.sh --target itms --noexec \
+	&& cd itms \
+	&& sed -e '15,43d' -i install_script.sh \
+	&& apt-get update \
+	&& apt-get install sudo \
+	&& ./install_script.sh
+
+FROM buildpack-deps:jessie as xar_builder
+
+ENV XAR_VERSION "1.6.1"
+USER root
+
+WORKDIR /tmp
+
+# Build xar
+ADD https://github.com/downloads/mackyle/xar/xar-$XAR_VERSION.tar.gz .
+RUN tar -xzf xar-$XAR_VERSION.tar.gz \
+	&& mv xar-$XAR_VERSION xar \
+	&& cd xar \
+	&& ./autogen.sh --noconfigure \
+	&& ./configure \
+	&& make 
+
+
+###############
+# Final image #
+###############
+FROM circleci/ruby:2.3
+MAINTAINER milch
+
+ENV PATH $PATH:/usr/local/itms/bin
+
+# Java versions to be installed
+ENV JAVA_VERSION 8u131
+ENV JAVA_DEBIAN_VERSION 8u131-b11-1~bpo8+1
+ENV CA_CERTIFICATES_JAVA_VERSION 20161107~bpo8+1
+
+# Needed for fastlane to work
+ENV LANG C.UTF-8
```

Comment:
Do you know what the `C` here means and why it is not `en_US` as on macOS? I still have an open support topic at Circle for that: https://discuss.circleci.com/t/difference-between-en-us-utf-8-and-c-utf-8/18835

Reply:
I believe for fastlane, it makes no difference, since it is language/region independent. 

IIRC the regular C locale is just ASCII as it is used by the C language, C.UTF-8 is therefore ASCII extended by UTF-8. I think generally both en_US.UTF-8 (or any other two letter language and two letter region combo) and C.UTF-8 should work for us. It is simply what is installed by default on Debian/Ubuntu, so rather than installing a separate locale I chose that one

Reply:
During the transition to Ubuntu I tried `en_US.` and got error messages as I didn't specify anything at all, changing to `C.` fixed this. But `en_US` not being installed makes sense.